File: 2710-remove-trailing-zeros-from-a-string/2710-remove-trailing-zeros-from-a-string.java
class Solution {
   public String removeTrailingZeros(String num) {
        int index = 0;
        for (int i = num.length() - 1; i >= 0; i--) {
            if (num.charAt(i) == '0') {
                index++;
            } else {
                break;
            }
        }
        
        String result = num.substring(0, num.length() - index);
        return result;
    }
}

File: 2710-remove-trailing-zeros-from-a-string/2710-remove-trailing-zeros-from-a-string.py
class Solution:
    def removeTrailingZeros(self, num: str) -> str:
        if num is None or num == "":
            return ""
        index = len(num) - 1
        while index >= 0 and num[index] == "0":
            index -= 1
        if index < 0:
            return "0"

        return num[: index + 1]
